### ISSUE TITLE
E116: Invalid arguments for function <SNR>251_addMapping2

### ISSUE THREAD

**Issue by sunaku** [wellle/targets.vim#191]
Hello,

After updating from 0f91613 to 7c6cf49, I get the following errors in NeoVim 0.2.0:

```
E121: Undefined variable: s:l
E116: Invalid arguments for function <SNR>251_addMapping2
line   13:
E121: Undefined variable: s:n
E116: Invalid arguments for function <SNR>251_addMapping2
line   14:
```

Here are the details of the update:

```
bundle/textobj/targets: Updating 0f91613..7c6cf49
bundle/textobj/targets: Fast-forward
bundle/textobj/targets:  README.md            |   38 +-
bundle/textobj/targets:  autoload/targets.vim |   10 +-
bundle/textobj/targets:  cheatsheet.md        |   32 +-
bundle/textobj/targets:  doc/targets.txt      |   74 +--
bundle/textobj/targets:  plugin/targets.vim   |   24 +-
bundle/textobj/targets:  test/test.vim        |    8 +-
bundle/textobj/targets:  test/test1.ok        | 1536 --------------------------------------------------
bundle/textobj/targets:  test/test1.out       | 1536 --------------------------------------------------
bundle/textobj/targets:  test/testM.ok        |    2 +-
bundle/textobj/targets:  test/testM.out       |    2 +-
bundle/textobj/targets:  10 files changed, 61 insertions(+), 3201 deletions(-)
```

Thanks for your consideration.

**Comment by sunaku** [comment]
Oh, I see that `g:targets_nlNL` was deprecated in 7c6cf49. 💡 Updating my config has fixed the issue:

```diff
diff --git a/bundle/textobj/targets.vim b/bundle/textobj/targets.vim
index f0da491..d985d28 100644
--- a/bundle/textobj/targets.vim
+++ b/bundle/textobj/targets.vim
@@ -1,2 +1,2 @@
 let g:targets_aiAI = 'ai  '
-let g:targets_nlNL = 'nN  '
+let g:targets_nl = 'nN'
```

**Comment by wellle** [comment]
@sunaku: Thanks for the quick heads up 👍 

Turns out I used Vimscript substring indexing wrongly. `g:targets_nlNL` is deprecated, but it should still be working as before. I was trying to extract the first two characters by accessing `[0:2]`, but this actually takes three characters. I just fixed that in #192.

Could you do me a favor and revert your config change and confirm that the current master works with that? Thanks!

**Comment by sunaku** [comment]
Confirmed: error no longer occurs in after reverting config.  Thanks.
